spec/models/whitehall_asset_spec.rb

```diff
@@ -44,6 +44,17 @@
           expect(asset).not_to be_valid
           expect(asset.errors[:legacy_url_path]).to include('is already taken')
         end
+
+        context 'but the existing asset has been deleted' do
```

Comment:
Minor: I think it might be clearer to say "marked as deleted" in the example name, rather than just "deleted".

Reply:
I've updated this and force pushed.